samples/nrf9160/modem_shell/src/link/link.c

```diff
@@ -247,6 +247,10 @@ void link_init(void)
 
 	lte_lc_register_handler(link_ind_handler);
 
+	if (link_sett_is_dnsaddr_enabled()) {
+		(void)link_setdnsaddr(link_sett_dnsaddr_ip_get());
+	}
+
```

Comment:
I was wondering if this is a correct place to set this as this is done only at boot up. But depends on wanted functionality, but usually these kind of functionalities are set always when going to normal mode.

Reply:
I think it's sufficient to set this only at boot up (and when changed), because it's a special DNS address configuration not bound to any PDN connection.

Reply:
ok